Glyssen/Block.cs

```diff
@@ -545,18 +545,41 @@ public string GetTextAsHtml(bool showVerseNumbers, bool rightToLeftScript)
 			return bldr.ToString();
 		}
 
+		public bool TryGetLeadingPunctuation(out ScriptText scriptTextWithLeadingPunctuation)
+		{
+			if (BlockElements.FirstOrDefault() is ScriptText initialScriptText &&
+			    BlockElements.Skip(1).FirstOrDefault() is Verse initialVerse &&
+			    initialVerse.Number == InitialVerseNumberOrBridge)
+			{
+				scriptTextWithLeadingPunctuation = initialScriptText;
+				return true;
+			}
+
+			scriptTextWithLeadingPunctuation = null;
+			return false;
+		}
+
 		public string GetSplitTextAsHtml(int blockId, bool rightToLeftScript, IEnumerable<BlockSplitData> blockSplits, bool showCharacters)
 		{
 			var bldr = new StringBuilder();
 			var currVerse = InitialVerseNumberOrBridge;
 			var verseNumberHtml = Empty;
-			const string splitTextTemplate = "<div class=\"splittext\" data-blockid=\"{2}\" data-verse=\"{3}\">{0}{1}</div>";
-
-			foreach (var blockElement in BlockElements)
+			var leadingPunctuationHtml = Empty;
+			const string splitTextTemplate = "<div class=\"splittext\" data-blockid=\"{2}\" data-verse=\"{3}\">{4}{0}{1}</div>";
+			const string leadingPunctuationTemplate = "<span class=\"leading-punctuation\">{0}</span>";
+
+			// Look for special case where verse has leading punctuation before the verse number such as
+			// ({1} This verse is surrounded by parentheses)
+			// This can only happen at the beginning of a block.
+			// If we have it, we basically want to do the split as if it wasn't there at all. i.e. Split the main part of the verse only, and
+			// do not include the leading punctuation as part of the offset.
+			if (TryGetLeadingPunctuation(out ScriptText initialScriptText))
+				leadingPunctuationHtml = Format(leadingPunctuationTemplate, initialScriptText.Content);
+
+			foreach (var blockElement in BlockElements.Skip(leadingPunctuationHtml != Empty ? 1 : 0))
```

Comment:
Interesting that you use a TryGet pattern here, apparently to avoid the cumbersomeness of having to check for a null return value, but then right below you have to do a check for Empty (or null if you make the change I suggest above).